Cargo.toml

```diff
@@ -20,11 +20,11 @@ resolver = "2"
 # We are using our own fork of `rust-dlc` at least until we can drop all the LN-DLC features. Also,
 # `p2pderivatives/rust-dlc#master` is missing certain patches that can only be found in the LN-DLC
 # branch.
-dlc-manager = { git = "https://github.com/get10101/rust-dlc", rev = "1010f95" }
```

Comment:
Should this be in this PR? Its not referred in the PR description is it?

Reply:
I can drop it since we rely on PR names for the change log.

Reply:
Done.